backend/pkg/api/connect/errors/common_errors.go

```diff
@@ -27,3 +27,14 @@ func NewRedpandaAdminAPINotConfiguredError() *connect.Error {
 		NewHelp(NewHelpLinkConsoleReferenceConfig()),
 	)
 }
+
+// NewRedpandaFeatureNotSupportedError is a standard error to return if an endpoint
+// requires the Redpanda feature that is not supported with current running version and instance.
+func NewRedpandaFeatureNotSupportedError(feature string) *connect.Error {
+	return NewConnectError(
+		connect.CodeUnimplemented,
+		errors.New("redpanda version does not support feature "+feature),
```

Comment:
We could put the feature into the metadata instead. I assume it's some sort of an enum?